### PR TITLE
Use variable for .content-box width

### PR DESCRIPTION
**Pull request by prtksxna**
From the comment on https://gerrit.wikimedia.org/r/#/c/422764/

### CODE REVIEW

**Comment by prtksxna** [comment]
Also from the gerrit patch that this is from, "Could you point me to the variable name convention that we're using here. I am not experienced with BEM and would like to learn the convention (if that is what we are following)"

File: css/wmui-style-guide.dev.css [hunk 1]
```diff
@@ -681,7 +683,7 @@ code {
 /* ... Tablet ... */
 @media ( min-width: 768px ) {
 	.content-box {
-		max-width: 960px;
+		max-width: var( --witdth-site );
```
Comment (Volker-E) [inline css/wmui-style-guide.dev.css]: typo! `width`

**Comment by prtksxna** [comment]
@Volker-E I am not sure if its github or something I am doing wrong: even after updating the PR it still says changes requested. Do you mind taking a look at thing again?

File: css/wmui-style-guide.dev.css [hunk 2]
```diff
@@ -750,6 +752,6 @@ code {
 
 @media ( min-width: 1200px ) {
 	.content-box {
-		width: 1140px;
+		width: var( --width-site-large );
```
Comment (Volker-E) [inline css/wmui-style-guide.dev.css]: Last rfc: In WikimediaUI Base there's a var `--width-breakpoint-desktop-wide`, this should be `--width-site-desktop-wide`.

Comment (prtksxna) [reply css/wmui-style-guide.dev.css]: 👍

**Comment by prtksxna** [comment]
Changed the var name. 

It seems that Github doesn't understand when commits are updated and keeps the status the same, in this case "Changes requested" will push more commits to the branch instead of updating the same patch from next time. We can just squash and merge in the end. Does that seem sensible @Volker-E?

**Comment by prtksxna** [comment]
> It seems that Github doesn't understand when commits are updated and keeps the status the same, in this case "Changes requested" will push more commits to the branch instead of updating the same patch from next time. We can just squash and merge in the end. Does that seem sensible @Volker-E?

Never mind this comment. I think I just don't understand how pull requests work.

**Comment by prtksxna** [comment]
Thanks @Volker-E

**Pull request merged**